ej2_streamlit_grids/frontend/src/styles/bootstrap-dark.css

```diff
@@ -0,0 +1,11 @@
+@import 'https://cdn.syncfusion.com/ej2/20.4.40/ej2-base/styles/bootstrap-dark.css';
```

Comment:
Why have you used these cdn link. You can use common ej2 cdn instead of this. No need to use these file.

Reply:
cdn link modified